src/main/frontend/components/settings.cljs

```diff
@@ -1069,6 +1092,7 @@
                 [:account "account" (t :settings-page/tab-account) (ui/icon "user-circle")])
                [:general "general" (t :settings-page/tab-general) (ui/icon "adjustments")]
                [:editor "editor" (t :settings-page/tab-editor) (ui/icon "writing")]
+               [:keymap "keymap" (t :settings-page/tab-keymap) (ui/icon "command")]
```

Comment:
"Command" icon is intuitive to Windows/Linux users. Is there any better icon for a "keyboard" or key?